src/Features/VisualBasic/Portable/Completion/CompletionProviders/ImplementsClauseCompletionProvider.vb

```diff
@@ -171,8 +171,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
             Next
 
             Dim symbols = semanticModel.LookupSymbols(position)
+            Dim availableInterfacesAndContainers = interfacesAndContainers.Where(Function(s) symbols.Contains(s.OriginalDefinition)).ToImmutableArray()
```

Comment:
Are symbols themselves guaranteed to be an OriginalDefinition? What happens if I'm something like B(Of T) Implements IList(Of T)? or perhaps IList(Of Integer) (as a concrete unrelated type)?